### DISCUSSION TITLE
Changing the font size only for specific show-text OSD item

### DISCUSSION THREAD

**Discussion by advice2020** [mpv-player/mpv#15070] [General]
If you use the following setting `--osd-font-size=<size>` this changes the font size of all OSD text elements.  I was wondering if there was a way to only change the font size for a specific `show-text` item only?  So for example if you have a `show-text` OSD set to a specific hotkey, that it would only change that items font size and not any others.

Thank You

**Comment by guidocella** [comment]
https://mpv.io/manual/master/#command-interface-osd-ass-cc

e.g. `show-text '${osd-ass-cc/0}{\fs20}foo'`
